influxql/functions.go

```diff
@@ -419,7 +425,7 @@ func ReduceStddev(values []interface{}) interface{} {
 		sq := math.Pow(dif, 2)
 		variance += sq
 	}
-	variance = variance / float64(len(data)-1)
+	variance = variance / float64(count-1)
```

Comment:
it probably wouldn't be a bad idea to add a test for `count < 2` though.  i can work on that if you want.